modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-extend-edge-nodes-aws-local-zones_{context}"]
+= Creating user workloads in AWS Local Zones
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create a user workloads in Local Zone subnets. 
+
+After the `openshift-installer` creates the cluster, the installer automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod to a node if the pod's deployment does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
```

Comment:
@mtulio . `a pod's deployment` correct? We create a `deployment` resource (a pod) that operates inside an edge worker node?

Reply:
The `deployment` is an abstract object that defines the pod, the `pod` will run on the 'edge worker nodes' with statement commented above (nodeSelectors and tolerations)

Reply:
Thanks. I updated this sentence to clarify the pod/deployment query:

> This means that a scheduler does not add a new pod, or deployment, to a node if the pod does not match the specified tolerations for a taint.